docs/specs/toc.yml

```diff
@@ -792,4 +792,22 @@ outputs:
   build.log: |
     ["error","invalid-toc-href","The toc href 'f1/a.md' can only reference to a local TOC file, folder or absolute path","docs/toc.yml"]
     ["error","invalid-toc-href","The toc href 'f1/a.md' can only reference to a local TOC file, folder or absolute path","docs/toc.yml"]
-
+---
+# Toc syntax error with duplicated keys
+inputs:
+  docfx.yml:
+  docs/toc.yml: |
+    - name: topic href has highest priority
+      topicHref: f1/a.md
+      topicHref: f1/a.md
+      tocHref: f1/
+      href: f1/b.md
+  docs/f1/a.md:
+  docs/f1/b.md:
+outputs:
+  docs/toc.json: 
```

Comment:
>json [](start = 11, length = 4)

what's the output toc looks like?

Reply:
the actual output toc is an empty list: 
{"items":[]}